apps/pipelines/nodes/nodes.py

```diff
@@ -74,9 +65,11 @@ def all_variables(in_):
 
 
 class LLMResponseMixin(BaseModel):
-    llm_provider_id: LlmProviderId
-    llm_provider_model_id: LlmProviderModelId
-    llm_temperature: LlmTemperature = Field(default=1.0, gt=0.0, le=2.0)
+    llm_provider_id: int = Field(..., title="LLM Model", json_schema_extra=UiSchema(widget=Widgets.llm_provider_model))
+    llm_provider_model_id: int = Field(..., json_schema_extra=UiSchema(widget=Widgets.none))
```

Comment:
Do we have to use the `Widgets.none` widget instead of `None` from now on?

Reply:
Depends what you want. `Widgets.none` will not display the field at all. If you want to use the default widget then you don't have to specify one and it will use one based on the field type.